vast/vast.cpp

```diff
@@ -33,6 +34,23 @@
 using namespace vast;
 using namespace vast::system;
 
+std::string format_error(caf::error err) {
+  std::ostringstream oss;
+  auto category = err.category();
+  oss << "Error: ";
+  if (category == caf::atom("vast"))
+    oss << to_string(static_cast<vast::ec>(err.code()));
+  else if (category == caf::atom("parser"))
+    oss << to_string(static_cast<caf::pec>(err.code()));
+  else if (category == caf::atom("system"))
+    oss << to_string(static_cast<caf::sec>(err.code()));
```

Comment:
There's no `else` here. So an error with a different category simply prints `"Error: "`. I think we can at least have a fallback to print the category atom.